job/job.go

```diff
@@ -70,7 +70,8 @@ func GetGenericJob(obj Object, config Config) (*batchv1.Job, error) {
 			Name:      obj.GetMetaObject().GetName(),
 			Namespace: obj.GetMetaObject().GetNamespace(),
 			Labels: map[string]string{
-				K8uplabel: "true",
+				K8uplabel:      "true",
+				"k8up.io/type": obj.GetType().String(),
```

Comment:
I suggest to extract this label into a const as well, as are `K8uplabel` and `K8upExclusive`.

Reply:
I was too lazy :D